cppcache/src/LocalRegion.cpp

```diff
@@ -717,7 +717,7 @@ void LocalRegion::registerEntryExpiryTask(
       new EntryExpiryHandler(rptr, entry, getEntryExpirationAction(), duration);
   auto id = rptr->getCacheImpl()->getExpiryTaskManager().scheduleExpiryTask(
       handler, duration, std::chrono::seconds::zero());
-  if (Log::finestEnabled()) {
+  if (Log::logLevel() >= LogLevel::Finest) {
```

Comment:
I am sort of indifferent on this change but the original was more closely aligned with the Java based logger API. This change requires you to know that log levels have order.

Reply:
Put `Log::enabled(LogLevel level)` back in, and am using that now.

Reply:
Added enabled() back, and using that now.  No need for warningEnabled, finestEnabled, etc helper functions tho.